app/main.py

```diff
@@ -3,3 +3,57 @@ def __init__(self, weight: int) -> None:
         self.weight = weight
 
 # write your code here
+
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
+
+    def go_forward(self, move: int = 1) -> None:
+        self.coords[1] += move
+
+    def go_back(self, move: int = 1) -> None:
+        self.coords[1] -= move
+
+    def go_right(self, move: int = 1) -> None:
+        self.coords[0] += move
+
+    def go_left(self, move: int = 1) -> None:
+        self.coords[0] -= move
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        super().__init__(name, weight, [0, 0, 0] if coords is None else coords)
```

Comment:
You could use `or` here, rewrite this code for simplicity
```suggestion
        super().__init__(name, weight, coords or [0, 0, 0])
```